samples/snippets/quickstart.py

```diff
@@ -0,0 +1,97 @@
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Google Analytics Data API sample quickstart application.
+Example usage:
+    python quickstart.py --property_id <PROPERTY_ID>
+
+    where <PROPERTY_ID> is the Google Analytics property id to use for a query.
+
+Note: you need to have the Google Analytics Data API enabled in your project
+prior to running this sample. Please visit the following URL and make sure the
+API is enabled:
+
+https://console.developers.google.com/apis/library/analyticsdata.googleapis.com
+
+This application demonstrates the usage of the Analytics Data API using
+service account credentials. For more information on service accounts, see
+
+https://cloud.google.com/iam/docs/understanding-service-accounts
+
+The following document provides instructions on setting service account
+credentials for your application:
+
+  https://cloud.google.com/docs/authentication/production
+
+In a nutshell, you need to:
+1. Create a service account and download the key JSON file.
+
+https://cloud.google.com/docs/authentication/production#creating_a_service_account
+
+2. Provide service account credentials using one of the following options:
+- set the GOOGLE_APPLICATION_CREDENTIALS environment variable, the API
+client will use the value of this variable to find the service account key
+JSON file.
+
+https://cloud.google.com/docs/authentication/production#setting_the_environment_variable
+
+OR
+- manually pass the path to the service account key JSON file to the API client
+by specifying the keyFilename parameter in the constructor:
+https://cloud.google.com/docs/authentication/production#passing_the_path_to_the_service_account_key_in_code
+
+To install the latest published package dependency, execute the following:
+  pip install google-analytics-data
+"""
+import argparse
+
+# [START ga_data_run_report]
+from google.analytics.data import AlphaAnalyticsDataAsyncClient
+from google.analytics.data_v1alpha.types import Entity
+from google.analytics.data_v1alpha.types import Dimension
+from google.analytics.data_v1alpha.types import Metric
+from google.analytics.data_v1alpha.types import DateRange
+
+def sample_run_report(property_id):
+  """Runs a simple report on a Google Analytics App+Web property."""
+
+  # Using a default constructor instructs the client to use the credentials
+  # specified in GOOGLE_APPLICATION_CREDENTIALS environment variable.
+  client = AlphaAnalyticsDataAsyncClient()
+  response = client.run_report(
+      Entity(property_id=property_id),
+      [Dimension(name='city')],
+      [Metric(name='activeUsers')],
+      [DateRange(start_date='2020-03-31', end_date='today')])
+
+  print("Report result:")
+  for row in response.rows:
+    print(row.dimension_values[0].value, row.metric_values[0].value)
+# [END ga_data_run_report]
+
+if __name__ == "__main__":
```

Comment:
If you want to simplify this, you can use environment variables.  Then you don't need this structure - you would just call the main function as a function and find the property id from the environment.  In the test you would do something like `os.environ["PROPERTY_ID"] = '123456'`

Reply:
Let's leave it as is for now so that we can publish samples, I'll followup with an update to externalize this property to kokoro config (for other platforms too).